api/test/integration/env/tools/print_active_agents.py

```diff
@@ -0,0 +1,3 @@
+from wazuh import agent
+
+print(agent.get_agents(select=['status'], filters={'status': 'active'})._affected_items)
```

Comment:
```suggestion
print(agent.get_agents(select=['status'], filters={'status': 'active'}).affected_items)
```

The `AffectedItemsWazuhResult` class has a property to access its `affected_items`. Avoid using protected attributes of a class.